packages/editor/src/components/inserter/menu.js

```diff
@@ -48,14 +49,30 @@ const MAX_SUGGESTED_ITEMS = 9;
  * @return {Array}             Filtered item list.
  */
 export const searchItems = ( items, searchTerm ) => {
-	const normalizedSearchTerm = searchTerm.toLowerCase().trim();
-	const matchSearch = ( string ) => string.toLowerCase().indexOf( normalizedSearchTerm ) !== -1;
+	const normalizedSearchTerm = normalizeTerm( searchTerm );
+	const matchSearch = ( string ) => normalizeTerm( string ).indexOf( normalizedSearchTerm ) !== -1;
 
 	return items.filter( ( item ) =>
 		matchSearch( item.title ) || some( item.keywords, matchSearch )
 	);
 };
 
+/**
+ * Converts the search term into a normalized term, removing diacritics,
+ * leading & trailing whitespace. The term is also converted to
+ * lowercase.
+ *
+ * @param {string} term The search term to normalize
+ * @return {string} The normalized search term
```

Comment:
Conventionally we're including a newline before the `@return` JSDoc tag, per documentation example at:

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#functions

Reply:
Also periods after each sentence.